src/pydvl/influence/__init__.py

```diff
@@ -1,10 +1,8 @@
 """
 This package contains algorithms for the computation of the influence function.
 
-.. warning::
-   Much of the code in this package is experimental or untested and is subject
-   to modification. In particular, the package structure and basic API will
-   probably change.
+> **Warning:** Much of the code in this package is experimental or untested and is subject to modification.
```

Comment:
Do we still need this warning even after the refactoring?

Reply:
I would like to keep it for now.